page/NotificationForm.jsx

```diff
@@ -1,118 +1,209 @@
-import { useState, useEffect, useRef } from 'react'
-import { Text, View, Button, Platform, TextInput } from 'react-native'
-import * as Device from 'expo-device'
-import * as Notifications from 'expo-notifications'
-import axios from 'axios'
+import { useState, useEffect, useRef } from 'react';
+import {
+	Text,
+	View,
+	Platform,
+	TextInput,
+	SafeAreaView,
+	StyleSheet,
+	TouchableOpacity,
+} from 'react-native';
+import * as Device from 'expo-device';
+import * as Notifications from 'expo-notifications';
+import axios from 'axios';
+import React from 'react';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
```

Comment:
You are importing from `react` twice. Add all imports in one line

```suggestion
import React, { useState, useEffect, useRef } from 'react';
import {
	Text,
	View,
	Platform,
	TextInput,
	SafeAreaView,
	StyleSheet,
	TouchableOpacity,
} from 'react-native';
import * as Device from 'expo-device';
import * as Notifications from 'expo-notifications';
import axios from 'axios';
import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
```